.github/autobuild/android.sh

```diff
@@ -70,7 +70,7 @@ setup_qt() {
         echo "Installing Qt..."
         python3 -m pip install "aqtinstall==${AQTINSTALL_VERSION}"
         python3 -m aqt install-qt --outputdir "${QT_BASEDIR}" linux android "${QT_VERSION}" \
-            --archives qtbase qttools qttranslations qtandroidextras
+            --archives qtbase qttools qttranslations qtandroidextras qtmultimedia
```

Comment:
I've had a quick look at `qtmultimedia` regarding the crash/dynamic linking problem. It looks like `qtmultimedia` contains a library (ibQt5MultimediaQuick_armeabi-v7a.so) which introduces a dependency on libQt5Quick_armeabi-v7a.so, which we do not download/package.

As a quick workaround, we could try to install that library as well. However, it sounds wrong to do so as it's QML-related (I think), which we do not use at all:

```suggestion
            --archives qtbase qttools qttranslations qtandroidextras qtmultimedia qtdeclarative
```

The real path forward should be getting rid of the unused dependency. I don't know if there are any magic flags to accomplish that. One thing we could try would be removing the unused libraries in order to avoid them being included in the APK:
```suggestion
            --archives qtbase qttools qttranslations qtandroidextras qtmultimedia
        # Delete libraries, which we don't use, but which bloat the resulting package and might introduce unwanted dependencies.
        find "${QT_BASEDIR}" -name 'libQt5*Quick*.so' -delete
        rm -r "${QT_BASEDIR}/${QT_VERSION}/android/qml/"
```
I haven't tried it, but I imagine that it could work.

Some logs from my testing:
```
# aqt install-qt --outputdir /tmp/x linux android 5.15.2 --archives qtmultimedia qtquick
# for x in /tmp/x/5.15.2/android/lib/libQt5*-v7a.so; do echo FILE $x; readelf -a $x | grep -P '^FILE|libQt5Quick_armeabi-v7a.so'; done
FILE /tmp/x/5.15.2/android/lib/libQt5MultimediaQuick_armeabi-v7a.so
 0x00000001 (NEEDED)                     Shared library: [libQt5Quick_armeabi-v7a.so]
FILE /tmp/x/5.15.2/android/lib/libQt5MultimediaWidgets_armeabi-v7a.so
FILE /tmp/x/5.15.2/android/lib/libQt5Multimedia_armeabi-v7a.so
```

Reply:
Are there alternatives to `qtmultimedia` that aren't as bundled - i.e. omit the unwanted dependencies?

Reply:
I don't think so. The root cause might be in `androiddeployqt`'s heuristic for finding dependencies, which is documented to be prone to false positives:

https://doc.qt.io/qt-5/deployment-android.html#dependencies-detection
https://doc.qt.io/qt-5/qmake-variable-reference.html#android-deployment-dependencies
> However, since run-time usage of plugins cannot be detected, there could be false positives

Seems like one can disable auto-detection, but that would require us to list all libraries in the correct order ourselves (`ANDROID_DEPLOYMENT_DEPENDENCIES`).

I guess I'd still give the library removal a chance.

Reply:
@hoffie thank you so much for the hard work and details provided. Tested adding the qtdeclarative in my own branch and with that in place the android no longer crashes and the audio alerts are working as well. I have just update the android.sh script in this PR as well to get this updated as well.

> I guess I'd still give the library removal a chance.

Will look into this next.

Reply:
@hoffie, you are a star! The removal of these unwanted dependencies is also resolving the crash on Android. I have just updated and pushed the latest update. Qtdeclarative is now removed and the lib deletion in place. It worked fine on my own branch so sure it works here as well.